packages/verification-vue-component/src/components/Verification/items/cards/CertifyingCard.vue

```diff
@@ -54,7 +54,7 @@
                           :has-unverified-issuer="hasUnverifiedIssuer"
                           :has-verified-issuer="hasVerifiedIssuer"
                           :document-registration-in-progress="registrationInProgress"
-                          :document-revoked="verificationItem.revoked"
+                          :document-revoked="revokeEvents.length > 0"
```

Comment:
Do you really need to calculate the revocation state?  Is it possible to use the precalculated revoked file result somehow?  I would like to avoid several layer of state calculation.

Reply:
I haven't checked the js logic for these root states yet, so I don't trust them currently. Either we need to implement the same logic also in JS at some point, or it will change, when we use wasm for the local verification.